### PR TITLE
[MIG] Migrated product_form_sale_link to v13.

### PR DESCRIPTION
**Pull request by sudhir-erpharbor**

### CODE REVIEW

**Comment by manuelcalerosolis** [comment]
Functional test ok!

**Comment by rousseldenis** [comment]
@sudhir-erpharbor Could you rebase ?

**Review by rousseldenis — APPROVED** [review]

**Comment by JSeguiN97** [comment]
@rousseldenis I follow this PR in #1529

File: product_form_sale_link/views/product_product.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            <button name="action_view_sales" position="after">
+                <button class="oe_stat_button" name="%(product_form_sale_link.action_product_product_sale_list)d"
+                    type="action" icon="fa-list">
+                    <field string="Sales" name="sales_count" widget="statinfo" />
+                </button>
+            </button>
+        </field>
+    </record>
+
+</odoo>
```
Comment (hailangvn) [inline product_form_sale_link/views/product_product.xml:23]: Please help to merge with origin/13.0 then `pre-commit run -a` and commit changes to this branch.

File: product_form_sale_link/views/product_template.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            <button name="action_view_sales" position="after">
+                <button class="oe_stat_button" name="%(product_form_sale_link.action_product_template_sale_list)d"
+                    type="action" icon="fa-list">
+                    <field string="Sales" name="sales_count" widget="statinfo" />
+                </button>
+            </button>
+        </field>
+    </record>
+
+</odoo>
```
Comment (hailangvn) [inline product_form_sale_link/views/product_template.xml:26]: Save as above.

File: product_form_sale_link/views/sale_order_line.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    </record>
+
+    <record id="action_product_product_sale_list" model="ir.actions.act_window">
+        <field name="name">Sale Order Lines</field>
+        <field name="res_model">sale.order.line</field>
+        <field name="context">{}</field>
+        <field name="domain">[('state', 'in', ['sale', 'done']), ('product_id', 'in', active_ids)]</field>
+    </record>
+
+</odoo>
```
Comment (hailangvn) [inline product_form_sale_link/views/sale_order_line.xml:21]: Save as above.

**Review by hailangvn — COMMENTED** [review]
Please help to merge with origin/13.0 then `pre-commit run -a` and commit changes to this branch.

**Comment by rafaelbn** [comment]
Hello @sudhir-erpharbor , please,  are you going to continue this PR? Thaks for answering. Regards.

**Comment by rousseldenis** [comment]
@sudhir-erpharbor Could you follow migration guide ? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch

**Comment by rousseldenis** [comment]
@sudhir-erpharbor We merged #1529 instead

**Pull request closed without merging**